style.css

```diff
@@ -16,29 +16,108 @@
 }
 
 .hamburger {
+  position: fixed;
+  z-index: 100;
   height: 50px;
   width: 100%;
   display: flex;
   justify-content: space-between;
   align-items: center;
-  background: transparent;
+  background: #fff;
 }
 
 .hamburger-img {
   margin-right: 35px;
   margin-top: 15px;
+  padding: 0;
+  width: auto;
+  z-index: 100;
+  background: transparent;
+  cursor: pointer;
+  border: none;
 }
 
 @media (max-width: 768px) {
-  .link-desktop,
   .desk-illustr1,
   .desk-illustr2,
   .desk-illustr3,
-  .works-desk {
+  .works-desk,
+  .hide1,
+  .hamburger-desktop {
     display: none;
   }
 }
 
+.open {
+  color: #0f27ff;
+  font-size: 25px;
+}
+
+.close {
+  display: none;
+  color: white;
+  font-size: 20px;
+}
+
+.closingIcon {
+  position: fixed;
+  top: 1rem;
+  right: 1rem;
+  padding: 4px;
+  margin-top: 5%;
+}
+
+.textdecoration {
+  text-decoration: none;
+}
+
+.link-desktop {
+  box-sizing: border-box;
+  position: fixed;
+  transform: translateY(-100%);
+  transition: transform 0.2s;
+  width: 100%;
+  height: 100%;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  z-index: 99;
+  background: #f7f7f9;
+  backdrop-filter: blur(5px);
+  color: #3a4a42;
+  list-style: none;
+  padding-top: 4rem;
+  margin-top: 0;
+}
+
+.links-style a {
+  display: block;
+  padding-left: 25px;
+  padding-top: 20px;
+  color: #3a4a42;
+  font-weight: 600;
+  font-size: 32px;
+  line-height: 44px;
+  text-decoration: none;
+}
+
+.links-style li {
+  margin-bottom: 20px;
+}
+
+.list_none {
+  list-style: none;
+}
+
+.showMenu {
+  transform: translateY(0);
+}
+
+.hide_y_scroll {
+  overflow-y: hidden;
+}
+
```

Comment:
- [OPTIONAL] Great job hiding the hamburger menu button and On the desktop size menu page, however, I noticed that extending the mobile page size to desktop size when the mobile menu is open prevents the page to scroll. Please fix this to make sure the page can't scroll only if we are in mobile menu size and disable it once we get the desktop size.

you can user the media query to fix this. eg:
```css
@media (max-width: 992px) {
    .hide_y_scroll {
      overflow-y: scroll;
    }
 }
```